api/metrics/v1/metrics.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package clutch.metrics.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/metrics/v1;metricsv1";
+
+import "validate/validate.proto";
+import "api/v1/annotations.proto";
+import "google/api/annotations.proto";
+
+service MetricsAPI {
+  rpc GetMetrics(GetMetricsRequest) returns (GetMetricsResponse) {
+    option (google.api.http) = {
+      post : "/v1/metrics/query"
+      body : "*"
+    };
+    option (clutch.api.v1.action).type = READ;
+  }
+}
+
+message Query {
+  // Prometheus expression query string
+  string expression = 1 [ (validate.rules).string.min_len = 1 ];
+  // inclusive
+  int64 start_time_ms = 2 [ (validate.rules).int64.gte = 0 ];
```

Comment:
When I looked at the protoc gen validate stuff, I didn't see a good way to make these required. For example, there is a case where people might want to get values before 1970, which would require the value to be negative. I could wrap the value in a message and make the message required but not sure if that was the way to go.

Reply:
I think this is okay and not sure how data pre 1970 could be useful for clutch use cases but maybe I am not familiar with many

Reply:
lol well yeah its probably not useful for clutch cases :). I guess I was just thinking about how there could potentially be a universe where people wanted something like that. But yeah I think you are right I don't think we need to worry about that.